src/Store/Manager.php

```diff
@@ -132,17 +141,23 @@ public function invalidateContent($ids): static
     }
 
     public function invalidateModels($models): void
+    {
+        $models->each(function (Autocache $model) {
+            $model->delete();
```

Comment:
@SylvesterDamgaard there is potentially a race condition here where we delete before the cache is cleared. Maybe we should delete inside the UrlInvalidated event?

Reply:
@ryanmitchell Yes. Let's make the change. :)

Reply:
Done